toolkit/tools/imagecustomizerapi/systemconfig.go

```diff
@@ -108,5 +109,30 @@ func (s *SystemConfig) IsValid() error {
 		}
 	}
 
+	if s.Overlays != nil {
```

Comment:
does the verity node depend on the overlays node? if so, is it possible to add cross validation?

Reply:
thx Lanze.